packages/moonstone/EditableIntegerPicker/EditableIntegerPicker.js

```diff
@@ -52,6 +53,9 @@ const EditableIntegerPickerBase = kind({
 		/**
 		 * The maximum value selectable by the picker (inclusive).
 		 *
+		 * The value should be evenly divisible by
```

Comment:
Going to tweak this to say 'the range between `min` and `max` should be divisible by `step`'